app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,39 @@
+<h2>Sign up</h2>
+
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+  <div class="field">
+    <%= f.label :name %><br />
+    <%= f.text_field :name, autofocus: true %>
+  </div>
+
+  <div class="field">
+    <%= f.label :surname %><br />
+    <%= f.text_field :surname %>
+  </div>
+
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %>
+    <% if @minimum_password_length %>
+    <em>(<%= @minimum_password_length %> characters minimum)</em>
+    <% end %><br />
+    <%= f.password_field :password, autocomplete: "new-password" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password_confirmation %><br />
+    <%= f.password_field :password_confirmation, autocomplete: "new-password" %>
+  </div>
+
+  <div class="actions">
+    <%= f.submit "Sign up" %>
+  </div>
+<% end %>
+
+<%= render "devise/shared/links" %>
```

Comment:
- [ ] @dev89s and @akos29 Great work adding the Signup form, however, the design doesn't match the wireframe given, below is a comparison, with highlights on the differences.


Wireframe Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://github.com/dev89s/blog-app/assets/61408860/4e92b14b-80d6-4537-b258-756a1865203d)  | !![image](https://github.com/dev89s/blog-app/assets/61408860/f907bff3-dda6-4ff5-9a1f-7113b9a5077a)